client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -346,6 +346,7 @@ class AllDomains extends Component {
 				label: translate( 'Status' ),
 				isSortable: true,
```

Comment:
I'm wondering if we should change this `isSortable: false` instead, in my tests, it disabled the sort feature completely and also removed the `mouse :hover` effect and it started to behave like the last two columns:

https://github.com/Automattic/wp-calypso/assets/1044309/9019b1ab-8414-410f-990e-9e653532ebc4




How do you see it @allilevine?

Reply:
I believe we don't want to remove the sort feature of the field 🤔

Reply:
Yeah I think there's still value in sorting the column with errors on top. That's the default sorting for the table. If you sort by something else, you can still click "Status" to sort by it again. 

It's tricky because Status isn't like the last columns OR the ascending/descending columns.

Reply:
Or maybe we're overcomplicating things 😜